src/ChosenSubjectsColumns.js

```diff
@@ -0,0 +1,14 @@
+
+// sloupce bude lepsi vybrat takto hardcoded
+// jak budou vypadat sloupce ve vybranych predmetech
+export const subjectColumns = [
```

Comment:
Zrovna tady bych to nechal v tom file `ChosenSubject.js`. Nemá smysl to separovat, když se to jinde nepoužije. A nemá cenu přemýšlet do budoucna.